README.md

```diff
@@ -3,3 +3,131 @@
 [![Hackage](https://img.shields.io/hackage/v/life-sync.svg)](https://hackage.haskell.org/package/life-sync)
 [![Build status](https://secure.travis-ci.org/kowainik/life-sync.svg)](https://travis-ci.org/kowainik/life-sync)
 [![MIT license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/kowainik/life-sync/blob/master/LICENSE)
+[![Stackage LTS](http://stackage.org/package/life-sync/badge/lts)](http://stackage.org/lts/package/life-sync)
+[![Stackage Nightly](http://stackage.org/package/life-sync/badge/nightly)](http://stackage.org/nightly/package/life-sync)
+
+`life-sync` is CLI tool that makes it easier to synchronize `dotfiles`
+repository with personal configs across multiple machines.
+
+## Motivation
+
+You might have some configuration files with different settings for your system.
+For example:
+
+1. Preferred settings for your editors (Spacemacs, Vim, etc.).
+2. Useful bash aliases and other miscellaneous shell settings.
+3. Git configuration.
+
+And much more! But sometimes you start working from a new fresh machine without
+your settings, like in these situations:
```

Comment:
without having your setting within touch (or near or on hand, I don't know the best, but probably it's better to point out on that).